### PR TITLE
Fixes to navigation bar

### PR DESCRIPTION
**Pull request by fang-helen**
Expanded the size of the navbar links so that clicking in an area around the text will also take you to the intended page.
Also moved the navigation bar generation to script.js instead of on each html file, so that it is uniform across pages.

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -191,16 +248,18 @@ const dummyText = "Suggested for you"; // TODO: come up with variety
  */
 async function getEvents(url, index, option) {
   const eventListElements = document.getElementsByClassName('event-list-container');
+  url = "/display-event.html"; // use this for now
```
Comment (0tpyrk) [inline src/main/webapp/script.js:262]: Single quotes

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -12,6 +12,63 @@
...
+  headerRight.className = 'header-right';
+
+  const myLink = document.createElement('a');
+  myLink.className = 'nav-item';
+  myLink.href = '/my-events.html';
+  myLink.innerText = 'My Events';
+  headerRight.appendChild(myLink);
+
+  const header = document.getElementsByClassName('header')[0];
+  header.textContent = "";
```
Comment (0tpyrk) [inline src/main/webapp/script.js:55]: Single quotes

File: src/main/webapp/script.js (hunk 1 above)
Comment (fang-helen) [reply src/main/webapp/script.js]: Oof, I keep forgetting

Comment (0tpyrk) [reply src/main/webapp/script.js]: No worries, you got it literally everywhere else 👍

**Pull request merged**